packages/core-sdk/src/types/resources/license.ts

```diff
@@ -77,23 +77,41 @@ export type RegisterPILResponse = {
 };
 
 export type RegisterCommercialUsePILRequest = {
+  /** The fee to be paid when minting a license. */
   defaultMintingFee: string | number | bigint;
+  /** The ERC20 token to be used to pay the minting fee and the token must be registered in story protocol. */
   currency: Address;
+  /**
+   * The address of the royalty policy contract.
+   * @default 0xBe54FB168b3c982b7AaE60dB6CF75Bd8447b390E
```

Comment:
we should add a comment, this value is not very helpful
```suggestion
   * Defaults to {@link https://docs.story.foundation/docs/liquid-absolute-percentage | LAP|} policy address if not provided.
```